types/index.d.ts

```diff
@@ -86,8 +92,11 @@ declare namespace vfile {
     /**
      * Convert contents of `vfile` to string.
      * @param encoding If `contents` is a buffer, `encoding` is used to stringify buffers (default: `'utf8'`).
+     * @remarks
+     * This can also be used in a browser context, this accepts the same values as Node's `BufferEncoding`,
+     * but does not directly reference the type for compatibility.
      */
-    toString: (encoding?: BufferEncoding) => string
+    toString: (encoding?: string) => string
```

Comment:
Would defining `BufferEncoding` be useful? 

```ts
type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex";
```

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a2bc1d868d81733a8969236655fa600bd3651a7b/types/node/globals.d.ts#L173-L174

Reply:
It would be, my caution is that different versions of node support different encodings
Node 14: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a2bc1d868d81733a8969236655fa600bd3651a7b/types/node/globals.d.ts#L174
Node 4: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a2bc1d868d81733a8969236655fa600bd3651a7b/types/node/v4/index.d.ts#L119

It looks like they're only added, not removed, so maybe we set the Node 14 ones?

Reply:
That sounds fine 👍 It will mean they'll need to kept up to date..

Should we indicate somewhere that `encoding` won't be used in a browser context?

Reply:
Updated in fde2417

> Should we indicate somewhere that `encoding` won't be used in a browser context?

That's where it gets tricky, it *could* be used in a browser context, it's just less likely.